packages/formation/js/index.js

```diff
@@ -1,11 +1,14 @@
 import loadAccordionHandler from './accordion';
+import addSidenavListeners from './sidenav';
 import domready from 'domready';
 
 if (
   document.readyState === 'complete' &&
   (document.readyState !== 'loading' && !document.documentElement.doScroll)
 ) {
   loadAccordionHandler();
+  addSidenavListeners();
 } else {
   domready(loadAccordionHandler);
```

Comment:
We can probably wrap this in a function, that way we aren't calling domready multiple times.

```
domready(() => {
    loadAccordionHandler();
    addSidenavListeners();
})
```